### ISSUE TITLE
Episodes from a feed return after delete

### ISSUE THREAD

**Issue by ndias** [gpodder/gpodder#1456]
I have some problems with this feed http://feeds.tsf.pt/Tsf-SinaisT, some of the episodes from this feed from June 2019 and bellow return after I delete the episodes.
This happens afterr an update with a new episode from the feed.
If I "update podcast" and there aren't a new episode the delete episodes are not showed.

Testing the feed here
https://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Ffeeds.tsf.pt%2FTsf-SinaisT#l6535

I can see the deleted episodes in line that says "title contains bad characters"

Is gpodder acting like this because of the bad characters?

**Comment by mhussaincov93** [comment]
hi,
for people like me who are blind could something be added that will be spoken via the screen reader over the podcast name?
you have the visual element all set and I was thinking of myself.
for podcasts that have guid issues
like
podcast name this podcast has guuid issues
or something?

**Comment by auouymous** [comment]
@mhussaincov93 I'm not sure how to implement that other than sending a notification. Any type of error while updating a feed displays the icon and appends error to tooltip. If youtube is unreachable and you have 100 subscriptions, they would all fail and generate 100 notifications. Although, it might be easier to generate a single notification when finished updating, and pack every error in it. Would that be of any use?

**Comment by ndias** [comment]
@auouymous yes I know about the uniq GUID, in the logs I have only one complain about duplicated.

"2023-01-12 07:42:24,504 [gpodder.model] WARNING: Discarded episode with non-unique GUID, contact the podcast publisher to fix this issue. [TSF - Sinais - Podcast] [Edição de 24 de Julho 2015 - O Marciano de Baião]
"
but from a few minutes ago the feed seems ok, the previous episodes from 2019 are gone, maybe someone from TSF or the feed manager are monitoring issues in gpodder :)

Nevertheless I think the problem is not about the GUID because this feed http://feeds.tsf.pt/Tsf-MundoDigital from the same radio, has the same problem and don't have any duplicated GUID's

**Comment by mhussaincov93** [comment]
hi,
@auouymous
yes that sounds good :)
:)
